src/lib/auto_code_surveys.py

```diff
@@ -0,0 +1,134 @@
+"""
```

Comment:
@ritazagoni This looks a bit odd - there's a massive doc comment at the top of the file full of source?

Reply:
@lukechurch Oh yes that was me changing strategy. Updated.